source/installation.txt

```diff
@@ -20,50 +25,142 @@ For instructions on installing Relational Migrator, see the following pages:
 
      - Description
 
+     - Recommended scenario
+
    * - Local
 
-     - Suitable for evaluation and proof of concept jobs. 
+     - Install Relational Migrator to a single machine that cannot 
+       be externally accessed.
 
-       For more information, see :ref:`Install on a Local Machine 
-       <local-machine-install>`. 
+       For more information, see :ref:`<local-machine-install>`.
 
-   * - Unattended server
+     - Evaluation and proof of concept jobs that run for less than four hours.
+    
+   * - Docker
 
-     - Suitable for smaller production jobs. 
+     - Install Relational Migrator in a way that is simple to replicate
+       and independent of the operating system.
+
+       For more information, see :ref:`<rm-install-docker>`.
+
+     - Evaluation and proof of concept jobs that run for less than four hours,
+       and do not run directly on a server or machine.
        
-       For more information, see :ref:`Install on an Unattended Server 
-       <unattended-server>`. 
 
-   * - Docker
+   * - Unattended server
 
-     - Suitable if your familiar with Docker and don't want to run 
-       Relational Migrator directly on a server or machine.
+     - Install Relational Migrator on an unattended server that binds Relational 
+       Migrator to an IP address and exposes it as a web application.
```

Comment:
[question]

Does it bind to a specific port as well? If so, consider mentioning this.